engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/views/register.html

```diff
@@ -15,17 +15,21 @@ <h2 translate>Content Host Registration</h2>
       </li>
       <li>
         <p translate>Install the pre-built bootstrap RPM:</p>
-        <pre><code>rpm -Uvh http://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
+        <pre><code>rpm -Uvh https://{{ noCapsulesFound ? katelloHostname : hostname(selectedCapsule.url) }}/pub/{{ consumerCertRPM }}</code></pre>
       </li>
       <li>
-        <p translate>Run subscription-manager in a console on the client host.  You may use an Activation Key to register:</p>
+        <p translate>Register using subscription-manager:</p>
+        <p translate>Using an Activation Key:</p>
         <pre><code>subscription-manager register --org="{{ organization.label }}" --activationkey="&ltActivation Key Name&gt"</code></pre>
-        <p translate>or authenticate with a username and password:</p>
+        <p translate>Using a username and password:</p>
         <pre><code>subscription-manager register --org="{{ organization.label }}" --environment="Library"</code></pre>
       </li>
       <li>
-        <p translate>Install katello-agent for remote actions and displaying errata information:</p>
+        <p translate>Install client package:</p>
+        <p translate>For remote actions and reporting package, errata & tracer information:</p>
         <pre><code>yum -y install katello-agent</code></pre>
+        <p translate>For only report package, errata & tracer information:</p>
+        <pre><code>yum -y install katello-host-tools</code></pre>
```

Comment:
Sorry for the churn, I just got to thinking more about this. I'd tend towards putting this one first to treat it as the primary mode of operation. Putting katello-agent second makes it seem like the optional or advanced use case which is where I think we want to position it.

Second thought, do you think we should put any info here about remote actions via qpid/gofer or to note that REX can be used for remote actions but that does not require the agent package? Just trying to use this a good opportunity to make the options clear to users given the impacts to infrastructure that using `katello-agent` can bring.

Reply:
I definitely think mentioning REX here is a good idea. Maybe with a link to the docs?

Reply:
That turns this page from a “quickstart” style guide to an “advanced”, here’s all the options style.

Initially I think a quickstart style fits better here but, I will have a think about how we can display all that info in an easy to consume format.

At the very least we can link to the docs.

Reply:
I'd be fine with docs links to prevent info overload. With now having multiple options for client tools, and those options providing quite different implications for infrastructure (e.g. using katello-agent can put stress on the system via qpid queues).